helix-core/src/main/java/org/apache/helix/HelixAdmin.java

```diff
@@ -516,8 +527,7 @@ void addStateModelDef(String clusterName, String stateModelDef, StateModelDefini
    * @param resourceName
    * @return customized for the resource
    */
-  CustomizedView getResourceCustomizedView(String clusterName, String resourceName,
-      String customizedStateType);
+  CustomizedView getResourceCustomizedView(String clusterName, String resourceName, String customizedStateType);
```

Comment:
Do you set the right style formatter? Helix code has its own formatter in open source code repo

Reply:
changed the formatter